src/components/Pressable/PressableWithFeedback.js

```diff
@@ -46,9 +46,11 @@ const PressableWithFeedback = forwardRef((props, ref) => {
                         setDisabled(props.disabled);
                         return;
                     }
-                    onPress.then(() => {
-                        setDisabled(props.disabled);
-                    });
+                    onPress
+                        .then(() => {
+                            setDisabled(props.disabled);
+                        })
+                        .catch(() => setDisabled(props.disabled));
```

Comment:
Why this is required?

Reply:
Can you pinpoint to any real case that I can reproduce?
Also if that's the case we should just use `finally` instead of then & catch.